src/express-middleware/sitemap.js

```diff
@@ -5,7 +5,7 @@ export const sitemap = function (req, res, next) {
   generateSitemap(req).then((sitemap) => {
     if (Buffer.isBuffer(sitemap)) {
       res.set('Content-Type', 'application/x-gzip');
-      res.set('Content-Encoding', 'gzip');
+      res.set('Content-Encoding', 'compress');
```

Comment:
@dobri1408 This isn't correct either. We need to remove the Content-Encoding header entirely. Because the gzip compression is supposed to remain after the file is downloaded, not only during transfer.